elasticsearch-scylla/README.md

```diff
@@ -0,0 +1,96 @@
+5. If you are using Linux, you will need [docker-compose](https://docs.docker.com/compose/install/).
+
+### Building the images
+```
+git clone https://github.com/scylladb/scylla-code-samples.git
+cd scylla-code-samples/elasticsearch-scylla
+docker-compose build
+```
+
+To use the Twitter app to mine data from Twitter, modify the twitter section of docker-compose.yml with your developer API credentials and desired Twitter topic.
+
+### Starting the containers
+```
+docker-compose up -d
+```
+
+Please wait about a minute for all the services to start properly.
+
+### Check the status of the containers
+```
+docker ps
+```
+
+You should see the following containers running:
+
+```
+CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS              PORTS                    NAMES
+b5e5cb9a536f        elasticsearchscylla_kibana                 "/bin/sh -c 'cd /k..."   49 seconds ago      Up 46 seconds       0.0.0.0:5601->5601/tcp   elasticsearchscylla_kibana_1
+a4a18e3ba26f        elasticsearchscylla_scylla-node3           "/bin/sh -c 'bash ..."   49 seconds ago      Up 46 seconds                                elasticsearchscylla_scylla-node3_1
+3214c275b49d        elasticsearchscylla_twitter                "/bin/sh -c 'npm i..."   49 seconds ago      Up 47 seconds                                elasticsearchscylla_twitter_1
+0d7f66c4b1c9        elasticsearchscylla_elasticsearch-slave1   "/bin/sh -c 'bash ..."   49 seconds ago      Up 46 seconds                                elasticsearchscylla_elasticsearch-slave1_1
+b5b04ac809b8        elasticsearchscylla_elasticsearch-master   "/bin/sh -c 'bash ..."   49 seconds ago      Up 45 seconds       0.0.0.0:9200->9200/tcp   elasticsearchscylla_elasticsearch-master_1
+f4e03f46a8ea        elasticsearchscylla_scylla-node2           "/bin/sh -c 'bash ..."   49 seconds ago      Up 47 seconds                                elasticsearchscylla_scylla-node2_1
+d2bfecf52f4d        elasticsearchscylla_scylla-node1           "/bin/sh -c 'bash ..."   49 seconds ago      Up 46 seconds                                elasticsearchscylla_scylla-node1_1
+```
+
+### Checking the Twitter data from Scylla with cqlsh
+```
+docker exec -it elasticsearchscylla_scylla-node1_1 cqlsh
+cqlsh> select * from twitter.tweets;
+```
+
+### Dumping the data from Scylla to Elasticsearch
+```
+curl http://127.0.0.1:8080/dump
+```
+Each time a dump is done, the data in Elasticsearch is deleted.
+
+### To stop the tweets from writing to Scylla
+```
+curl http://127.0.0.1:8080/stop
+```
+
+### To start writing the tweets to Scylla again
+```
+curl http://127.0.0.1:8080/start
+```
+
+### Accessing Kibana to view the Twitter data
+1. Goto https://0.0.0.0:5601 in your web browser
+2. Enter logstash for the index name and then click the create button
```

Comment:
I do not see logstash in the dashboard. This is what I get
![image](https://user-images.githubusercontent.com/170200/28248489-fbf4a4c4-6a4d-11e7-9a62-8f7961416f64.png)

All docker instances are up and running
```
$ sudo docker ps
CONTAINER ID        IMAGE                                      COMMAND                  CREATED             STATUS                           PORTS                    NAMES
81526c2c6a62        elasticsearchscylla_twitter                "/bin/sh -c 'node ..."   9 minutes ago       Up 8 minutes                     0.0.0.0:8080->8080/tcp   elasticsearchscylla_twitter_1
b249ac3af2cc        elasticsearchscylla_kibana                 "/bin/sh -c 'cd /k..."   9 minutes ago       Up 8 minutes                     0.0.0.0:5601->5601/tcp   elasticsearchscylla_kibana_1
2a0fb4a55e43        elasticsearchscylla_scylla-node1           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node1_1
5f5ecdbd59a9        elasticsearchscylla_elasticsearch-master   "/bin/sh -c 'bash ..."   9 minutes ago       Up 1 second                      0.0.0.0:9200->9200/tcp   elasticsearchscylla_elasticsearch-master_1
cbee1fc66495        elasticsearchscylla_scylla-node3           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node3_1
8f3103e9f1cb        elasticsearchscylla_elasticsearch-slave1   "/bin/sh -c 'bash ..."   9 minutes ago       Restarting (78) 59 seconds ago                            elasticsearchscylla_elasticsearch-slave1_1
f72df4b33ffc        elasticsearchscylla_scylla-node2           "/bin/sh -c 'bash ..."   9 minutes ago       Up 8 minutes                                              elasticsearchscylla_scylla-node2_1
```